SS14.Launcher/Theme/Theme.xaml

```diff
@@ -80,6 +80,17 @@
       <Setter Property="TextAlignment" Value="Center" />
       <Setter Property="HorizontalAlignment" Value="Center" />
     </Style>
+    <Style Selector="^ TextBlock.Mono">
+      <Setter Property="FontFamily" Value="Mono" />
+      <Setter Property="TextAlignment" Value="Right" />
+      <Setter Property="HorizontalAlignment" Value="Right" />
```

Comment:
Doesn't make sense for these alignment properties to be here if the style class is just "Mono".

Reply:
That’s fair, it evolved to be used for numeric styles cells in the table but I didn’t update the name.